src/Order/Dto/AddGiftCardToCartInformationInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusGiftCardPlugin\Order\Dto;
```

Comment:
I think the namespace for this should be alongside the command. But that's just an opinion